spec/dummy/app/views/home/color.rb

```diff
@@ -0,0 +1,104 @@
+class Views::Home::Color < Views::Page
+  def main
+    h2 'Color'
+
+    guide '',
+      %{<strong>Do</strong> supplement color with text and icons to differentiate data or call out a UI element.}.html_safe,
+      '',
+      %{<strong>Don&#39;t</strong> attempt to communicate something to users with only a change in color. This is inaccessible to color-blind users.}.html_safe
+
+
+    h3 'Grayscale'
+
+    grayscaleColors.each do |var, color|
+      render('components/swatch', color: color, var: var)
+    end
+
+    hr
+
+    h3 'UI Colors'
+
+    p %{Each app should override <code>$primaryColor</code> and <code>$secondaryColor</code> with its primary brand colors.}.html_safe
+    uiColors.each do |var, color|
+      render('components/swatch', color: color, var: var)
+    end
+
+    hr
+
+    h3 'Label Colors'
+    p 'When building a feature that allows users to add labels to their data, use this color palette for the suggested presets.'
+
+    labelColors.each do |var, color|
+      render('components/swatch', color: color, var: var)
+    end
+
+    hr
+
+    h3 'Qualitative Colors'
+    p 'Use this color range to provide a value judgment (ranging from &ldquo;bad&rdquo; to &ldquo;good&rdquo;) on user-generated information.'.html_safe
+
+    qualColors.each do |var, color|
+      render('components/swatch', color: color, var: var)
+    end
+
+    hr
+  end
+
+  def uiColors
+    {
+      '$primaryColor' => '#697182',
```

Comment:
It kinda sucks that we're duplicating these, huh? Let's at least add a code comment to alert maintainers that they need to be kept in-sync...